code/style.css

```diff
@@ -1,7 +1,332 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+/* Removing default margins and boarder for all document */
+*{
+  margin: 0;
+  padding: 0;
+  border: 0 none;
+}
+
+/* Implementing box-sizing model */
+*, *::before, *::after{
+  box-sizing:border-box;
+}
 
 .active + .answer {
-  /* some styling here */
+  display:block;
+}
+
+/* -------- THE DESIGN OF SITE IS MADE MOBILE FIRST---------- */
+
+
+/* following styling affects the header section */
+.header{
+  width:100%;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  border-bottom: 2px solid black;
+}
+
+.navigation{
+  display: flex;
+  justify-content: space-around;
+  padding: 20px 10px;
+  display:none;
+  
+}
+        .navigation-link{
+          font-size: 20px;
+          color: white;
+          font-size: 14px;
+          font-family: 'Permanent Marker', cursive;
+        }
+
+
+.hamburger {
+  display: flex;
+  height: 18px;
+  width: 20px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 1.5rem;
+  z-index: 5;
+}
+
+      .hamburger div{
```

Comment:
probably a just a preference thing, but perhaps move the hamburger div to the left  so the code looks cleaner.